steps/virtual-device-testing-for-android/0.9.1/step.yml

```diff
@@ -0,0 +1,165 @@
+      For example:
+      `NexusLowRes,24,en,portrait`
+      `NexusLowRes,24,en,landscape`
+    is_required: true
+    summary: |
+      One device configuration per line and the parameters are separated with "," in the order of: "deviceID, version, language, orientation". For example: `NexusLowRes,24,en,portrait`
+    title: Test devices
+  test_devices: NexusLowRes,24,en,portrait
+- opts:
+    description: |
+      The type of your test you want to run on the device.
+    is_required: true
+    summary: The type of your test
+    title: Test type
+    value_options:
+    - instrumentation
+    - robo
+    - gameloop
+  test_type: instrumentation
+- app_package_id: null
+  opts:
+    description: |
+      The package ID of your app
+    summary: The package ID of your apk
+    title: App package ID
+- opts:
+    category: Instrumentation Test
+    description: |
+      The path to the unaligned test debug APK.
+    summary: The path to the unaligned test debug APK.
+    title: Test APK path
+  test_apk_path: null
+- inst_test_package_id: null
+  opts:
+    category: Instrumentation Test
+    description: |
+      The package ID of your test apk
+    summary: The package ID of your test apk
+    title: Test package ID
+- inst_test_runner_class: null
+  opts:
+    category: Instrumentation Test
+    description: |
+      The test runner that will run your test cases
+    summary: The test runner that will run your test cases
+    title: Test runner class
+- inst_test_targets: null
+  opts:
+    category: Instrumentation Test
+    description: |
+      Test targets
+    summary: Test targets
+    title: |
+      Test targets, seperated with the "," character.
+- opts:
+    category: Robo Test
+    title: Initial activity
+  robo_initial_activity: null
+- opts:
+    category: Robo Test
+    title: Max depth
+  robo_max_depth: null
+- opts:
+    category: Robo Test
+    title: Max steps
+  robo_max_steps: null
+- opts:
+    category: Robo Test
+    description: |
+      Test targets
+    summary: |
+      One directive per line, the parameters are separated with "," character. For example: "ResourceName,InputText,ActionType"
+    title: Robo directives
+  robo_directives: null
+- loop_scenarios: null
+  opts:
+    category: Game Loop Test
+    title: Loop scenarios
+- loop_scenario_labels: null
+  opts:
+    category: Game Loop Test
+    title: Loop scenario labels
+- opts:
+    category: Debug
+    summary: Maximum time allowed for the tests to run
+    title: Test timeout
+  test_timeout: 900
+- directories_to_pull: null
+  opts:
+    category: Debug
+    summary: Directories to pull, one path per line
+    title: Directories to pull
+- environment_variables: null
+  opts:
+    category: Debug
+    summary: |
+      Environment Variables, one per line and separated by "="
+    title: Environment Variables
+- download_test_results: false
+  opts:
+    category: Debug
+    is_required: true
+    title: Download test result files
```

Comment:
is there a case when we do not need the test result file? what kind of file are these?

Reply:
The step wont download files that are shown on the dashboard, however if user has own files created runtime(like codecov) then he can download them